packages/recoil/core/Recoil_RecoilRoot.js

```diff
@@ -383,17 +383,17 @@ function RecoilRoot_INTERNAL({
 
   let storeStateRef: {current: StoreState}; // eslint-disable-line prefer-const
 
-  const getGraph = (version: StateID) => {
+  const getGraph = useMemo((version: StateID) => {
```

Comment:
AFAIK useMemo's first argument is a function that takes no arguments, here it looks like you wanted to use a useCallback maybe?

Reply:
> AFAIK useMemo's first argument is a function that takes no arguments, here it looks like you wanted to use a useCallback maybe?

:)